synapse/tests/test_tools_superhash.py

```diff
@@ -0,0 +1,102 @@
+from synapse.tests.common import *
+
+import synapse.tools.superhash as s_superhash
+
+
+class SuperhashTest(SynTest):
+    @classmethod
+    def setUpClass(cls):
+        cls.expected_data = [
```

Comment:
Due to dictionaries being mutable, this data structure should be constructed separately to prevent any one test from modifying the data being used by another ( just make something like getIngData() that returns it...